numpy/core/tests/test_multiarray.py

```diff
@@ -6726,6 +6727,30 @@ def test_null_inside_ustring_array_is_truthy(self):
         a[0] = ' \0 \0'
         self.assertTrue(a)
 
+
+class TestCTypes(TestCase):
+    def setUp(self):
+        self.test_arr = np.array([[1, 2, 3], [4, 5, 6]])
```

Comment:
It's good practice to put cheap setup (like this) in the tests themselves, so that its self contained